### DISCUSSION TITLE
How to hide tooltip arrow?

### DISCUSSION THREAD

**Discussion by andrew-tacklit** [ant-design/ant-design#40036] [Q&A] [answered]
Hi, may I know how to hide the tooltip arrow? Thank you 🙏

![image](https://user-images.githubusercontent.com/110362852/210730389-e3b90d6d-587a-4d16-a518-f6e558092ac0.png)

**Comment by MadCcc** [comment, marked as answer]
Not a stable feature now, but you can use `showArrow={false}` to hide. Ref: https://github.com/react-component/tooltip
